src/allshoes.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import "./App.css";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCircle } from "@fortawesome/free-solid-svg-icons";
+import { Slide } from "react-awesome-reveal";
+
+export default function Allshoes(props) {
+  const { AllshoesName, AllImage } = props;
+  return (
+    <>
+      <Slide triggerOnce className="slide">
+        <div className="container">
+          <div className="listContainer">
+            {AllshoesName.map((ele) => {
+              return (
+                <div className="infoCard">
+                  <img
+                    alt="shoes"
+                    className="smalimg"
+                    width="100px"
+                    height="65px"
```

Comment:
Better to style size with CSS than in your HTML, to make it easier to customize.